modules/misc/news.nix

```
@@ -1880,6 +1880,17 @@ in
           https://no-color.org/.
         '';
       }
+      {
+        time = "2021-03-30T10:28:12+00:00";
+        condition = config.services.sxhkd.enable;
```

I think the reason this is failing is because the condition needs a check to ensure that it's on Linux:

```suggestion
        condition = hostPlatform.isLinux && config.services.sxhkd.enable;
```

---

Indeed, that is almost certainly the case.